### PR TITLE
Run irqbalance tests in serial within the actual suite

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
Running the tests in a serial fashion within the parallel suite.
#### Which issue(s) this PR fixes:

Fixes https://github.com/cri-o/cri-o/issues/7095

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
LGTM, but [test seems to be failing](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/cri-o_cri-o/7100/pull-ci-cri-o-cri-o-main-ci-fedora-integration/1674705185760350208/artifacts/fedora-integration/cri-o-gather/artifacts/testout.txt)

**Comment by saschagrunert** [comment]
We're still using bats 1.6.0 in the prow CI :thinking:

**Comment by saschagrunert** [comment]
Refers to #7113

**Review by littlejawa — APPROVED** [review]
lgtm, thanks!

**Comment by saschagrunert** [comment]
> LGTM, but [test seems to be failing](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/cri-o_cri-o/7100/pull-ci-cri-o-cri-o-main-ci-fedora-integration/1674705185760350208/artifacts/fedora-integration/cri-o-gather/artifacts/testout.txt)

Problem was that other tests override the global config and restore it on startup. This means we have to invert the logic of all tests and apply sane defaults.

Should work now, PTAL @haircommander 

/unhold

File: test/test_runner.sh [hunk 1]
```diff
@@ -35,8 +35,7 @@ TESTS=("${@:-.}")
 # The number of parallel jobs to execute tests
 export JOBS=${JOBS:-$(nproc --all)}
 
+bats --version
+
```
[the remark this answers is not in the document]
Comment (saschagrunert) [reply test/test_runner.sh:39]: Yes, we forgot to update the bats version in one of the CI scenarios which massively confused me. I think we can keep it.

**Pull request merged**